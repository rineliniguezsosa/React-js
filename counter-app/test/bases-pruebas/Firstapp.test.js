describe('Pruebas en el componente <Firstapp>', () => { 
  
 })