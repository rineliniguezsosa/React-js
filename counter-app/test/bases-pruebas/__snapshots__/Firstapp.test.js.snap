// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <Firstapp> Deberia de coincidir con el snapshot 1`] = `
<div>
  <span
    data-testid="prueba"
  >
    rinel
  </span>
  <span>
    27
  </span>
</div>
`;
